src/govuk/components/_all.scss

```diff
@@ -29,6 +28,7 @@
 @import "radios/index";
 @import "select/index";
 @import "skip-link/index";
+@import "summary-list/index";
```

Comment:
Thanks for tidying this up! I was surprised this was out of order and has been for so long – I _reckon_ this is because this component was originally called 'data list' – when we renamed it during development I suspect we didn't change its order in this file.